tools/realsense-viewer/CMakeLists.txt

```diff
@@ -15,9 +15,9 @@ find_package(Threads REQUIRED)
 
 include_directories(${CMAKE_BINARY_DIR})
 
-if(${BUILD_EASYLOGGINGPP})
-    include("${REPO_ROOT}/include/librealsense2/utilities/easylogging/easyloggingpp.cmake")
-endif()
+include("${REPO_ROOT}/include/librealsense2/utilities/easylogging/easyloggingpp.cmake")
+target_sources( ${PROJECT_NAME} PRIVATE ${ELPP_SOURCE_FILES} )
+source_group( "EasyLogging++" FILES ${ELPP_SOURCE_FILES} )
```

Comment:
This line is found inside `easyloggingpp.cmake`, it's not working?

Reply:
I had a bunch of problems we the two (elpp and concurrency). So all three lines made it in.
I'll investigate to make it better in next PR if acceptable.

Reply:
Ideally, I'd like to do the same for `utilities` that I did for `realdds` -- move it into a library...

Reply:
Next PR is OK.. regarding `utilities` let's leave it for now :)

Reply:
The line here does NOT appear in the easyloggingpp.cmake!
It's hard to see (for me, too) but the cmake does only the headers, not the sources. This here adds the sources.

I can muck around with it if you want, but I think leave it alone I ran into a lot of trouble with crashes in the python (because static variable initialization ordering) when I tried to get things right.

Reply:
OK, let's leave it for mow and see later on